content/community/gsoc-2022.en.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: XSF and Google Summer of Code 2022
+---
+
+[![](images/logos/GSoC_2022_Logo.png)](XSF and GSoC 2022 Logo)
+
+The XMPP Standards Foundation (XSF) is planning to participate as a hosting organisation at the [Google Summer of Code 2022](https://summerofcode.withgoogle.com/).
+If you are interested to participate as student (or simply newcomer to open-source), mentor or XMPP project this is the page to start with!
+ 
+If you can imagine to part time (or even full time) on an open-source software project in the summer, then consider to join an XMPP real-time communication technology project and participate in the Google Summer of Code 2022!
```

Comment:
If you can imagine yourself part time (or even full time) working on an open-source software project during the summer, ...